config/src/config/execution_config.rs

```diff
@@ -45,7 +45,7 @@ impl Default for ExecutionConfig {
         ExecutionConfig {
             genesis: None,
             genesis_file_location: PathBuf::new(),
-            service: ExecutionCorrectnessService::Local,
+            service: ExecutionCorrectnessService::Thread,
```

Comment:
I will admit I'm a little reserved about this one.... I think flipping it now to evaluate is okay, but we should revisit where we are in a few weeks.

Reply:
What's wrong with this than local? At least this one includes ser/de and I think `safety_rules` uses it too.